src/AbstractFixer.php

```diff
@@ -89,21 +89,13 @@ public function supports(\SplFileInfo $file)
         return true;
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function configure(array $configuration = null)
     {
         if (!$this instanceof ConfigurationDefinitionFixerInterface) {
-            throw new \LogicException('Cannot run method for class not implementing `ConfigurationDefinitionFixerInterface`.');
+            throw new \LogicException(sprintf('Cannot configure using Abstact parent, child not implementing "%s".', ConfigurationDefinitionFixerInterface::class));
         }
 
         if (null === $configuration) {
-            @trigger_error(
```

Comment:
OK, so.
Actually, I thought that we could split one method of master branch - `configure`, which sometimes does the configuration and sometimes removes it, into 2 methods - `configure` and `reset`.
I thought that we would not run `resolver` when one want to reset the configuration.

But since we keep the default config in resolver we would call it in `reset` method anyway. For that, indeed it will be the same as `configure` method, so no need for 2nd method indeed.

then, maybe indeed passing `null` should be deprecated